### PR TITLE
Ruby on Rails capstone project - Budget app

### PR DESCRIPTION
**Pull request by benshidanny11**
### In this milestone I implemented:
- Splash screen
- Sign up and login pages
- Home page (categories page)
- Transactions page
- Add a new category page
- Add a new transaction page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,43 @@
-# README
+# Project: Budjet App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Built With
```
Comment (Davidosky007) [inline README.md:3]: * Kindly add a live version link of your project as it's [required](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.) that The project should be deployed and accessible online.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,43 @@
...
-* How to run the test suite
+- GitHub: [@benshidanny11](https://github.com/bensidanny11)
+- Twitter: [@DBenshi](https://twitter.com/DBenshi)
+- LinkedIn: [Daniel Urimubenshi](https://www.linkedin.com/in/danielurimubenshi/)
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🤝 Contributing
 
-* Deployment instructions
+Contributions, issues, and feature requests are welcome!
```
Comment (Davidosky007) [inline README.md:42]: * Kindly add to your README  an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). [Check requirement here ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)

File: app/views/groups/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<header class="bg-primary p-2 d-flex justify-content-around align-items-center">
```
Comment (Davidosky007) [inline app/views/groups/index.html.erb]: * Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. as currently you are using a different color here

File: app/views/groups/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<header class="bg-primary p-2">
```
Comment (Davidosky007) [inline app/views/groups/new.html.erb]: * Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. As your color here is different

File: app/views/operations/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<header class="bg-primary p-2 d-flex justify-content-around align-items-center">
```
Comment (Davidosky007) [inline app/views/operations/index.html.erb]: * Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. As your color and font are different

File: app/views/operations/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<header class="bg-primary p-2 d-flex justify-content-around align-items-center">
```
Comment (Davidosky007) [inline app/views/operations/new.html.erb]: * Kindly follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. As your color and font are  different

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @benshidanny11 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* All tests passed ✔️ 
* Descriptive commit messages 👍 
## Required Changes ♻️
* Kindly Note that Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

> a description of the project.
> a demo of the project features.
> you should also highlight some interesting piece of code or something you built that you are very proud of.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @benshidanny11 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**